packages/nextjs/tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "skipLibCheck": true,
     "sourceMap": false,
     "strict": true,
-    "target": "ES2020"
+    "target": "ES2020",
+    "rootDir": "src"
```

Comment:
❓ Should we also this to all the other packages of the monorepo? Is this change resolving some issue or something?

Reply:
No reason to :)